gemrb/GUIScripts/bg2/CharGenEnd.py

```diff
@@ -32,6 +32,22 @@ def OnLoad():
 	# set my character up
 	MyChar = GemRB.GetVar ("Slot")
 
+	#reputation
+	# If Rep > 0 then MyChar is an imported character with a saved reputation,
+	# in which case the reputation shouldn't be overwritten.
+	Rep = GemRB.GetPlayerStat (MyChar, IE_REPUTATION)
+
+	if Rep <= 0:
+		# use the alignment to apply starting reputation
+		RepTable = GemRB.LoadTable ("repstart")
+		AlignmentAbbrev = CommonTables.Aligns.FindValue (3, Alignment)
```

Comment:
Alignment will be undefined here, you need to copy that line as well.

Reply:
Thanks, didn't catch.